### PR TITLE
[zh]resync content/zh/docs/reference/command-line-tools-reference/kub…

### PR DESCRIPTION
**Pull request by howieyuen**
close: https://github.com/kubernetes/website/issues/31459

### CODE REVIEW

**Comment by howieyuen** [comment]
cc @mengjiao-liu

File: content/zh/docs/reference/command-line-tools-reference/kubelet.md [hunk 1]
```diff
@@ -1694,9 +1744,11 @@ Minimum age for an unused image before it is garbage collected.  Examples: `300m
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-IP address of the node. If set, kubelet will use this IP address for the node
+IP address (or comma-separated dual-stack IP addresses) of the node. If unset, kubelet will use the node's default IPv4 address, if any, or its default IPv6 address if it has no IPv4 addresses. You can pass <code>'::'</code> to make it prefer the default IPv6 address rather than the default IPv4 address.
 -->
-节点的 IP 地址。如果设置，kubelet 将使用该 IP 地址作为节点的 IP 地址。
+节点的IP地址（或逗号分隔的双栈IP地址）。
```
Comment (mengjiao-liu) [inline content/zh/docs/reference/command-line-tools-reference/kubelet.md:1749]:
```suggestion
节点的 IP 地址（或逗号分隔的双栈 IP 地址）。
```

**Pull request merged**